docs/reference/manifest/runtimes.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Runtimes in the manifest file
+description: ''
+ms.date: 01/06/2020
+localization_priority: Normal
+---
+
+# Runtimes element
+
+This feature is in preview. Specifies the runtime of your add-in and allows custom functions and the task pane to share global data and make function calls into each other. Should follow the `<Host>` element in your manifest file.
```

Comment:
Nit: Should "Specifies the runtime …" be "Specifies the runtimes …" since the element name is plural (Runtimes)?